src/common/parse.js

```diff
@@ -282,7 +282,8 @@ class ParseHelper {
     const results = await query.skip(skipCount).limit(fetchCount).find();
     const transactions = _.map(results, (item) => {
       const transaction = parseDataUtil.getTransaction(item);
-      return transaction;
+      const isSender = address === transaction.from;
```

Comment:
Since you are passing in the full transaction object into parseDataUtil.processTransaction(), why do you need to determine isSender here? You can determine it inside processTransaction() and don't need to pass it in.